jolly-jellyfish/src/page_maker/fixtures/templates.yaml

```diff
@@ -0,0 +1,14 @@
+- model: page_maker.template
+  pk: 1
+  fields:
+    name: hotdog
+    date_created: 2020-08-02 16:22:00+00:00
+    author: 1
+    style_sheet: styles/test.css
```

Comment:
Would it make more sense to have one of the included themes here?

Reply:
Pretty sure these files don't exist to begin with:

https://github.com/Artemis21/summer-code-jam-2020/pull/25#discussion_r467231618

Reply:
Oh, ok. @baru64 can you explain why?

Reply:
We only test the behavior of view, what objects does it return, what is the status code and when we check if there is an image or file in a page we only check if there is a link to it, we don't check if you can download the image.
To use actual files we can change the MEDIA_ROOT for tests to something like /test_media/ but there's not point in this unless we want to test downloading.

Reply:
Oh, of course, I thought this was what we discussed about including some articles default.